EventsExpress/ClientApp/src/services/EventsExpressService.js

```diff
@@ -105,6 +105,98 @@ export default class EventsExpressService {
         return res;
     }
 
+    setEvent = async (data) => {
```

Comment:
This code duplicates the doce from `setEventFromParent`. Consider extracting it in separate method and reusing it.

Reply:
Solved